### ISSUE TITLE
Ruby Dockerfile incorrectly references environment variables not already defined

### ISSUE THREAD

**Issue by rcousens** [aws/aws-codebuild-docker-images#64]
I haven't checked other images but in Ruby 2.5.1 you create environment variables in a single ENV line (using the multi-line separator) that is self-referential.

I.e. you have ENV FOO="BAZ" BAR=$FOO which won't work as BAR ends up being unset.

It has to be:

```
ENV FOO="BAZ"
ENV BAR=$FOO
```

This error broke bundler for me in the container. Setting the BUNDLE environment variables appropriately in the CodeBuild project rectified the issue.

I am happy to submit a pull request if you agree this is an issue that should be fixed. I think it should be!

The specific location of the problem is this part where BUNDLE environment variables reference GEM_HOME:

[aws-codebuild-docker-images/ubuntu/ruby/2.5.1/Dockerfile](https://github.com/aws/aws-codebuild-docker-images/blob/898a70fea06903dc0219caed79f9ad1c548b84f6/ubuntu/ruby/2.5.1/Dockerfile#L97)

**Comment by clareliguori** [comment]
Thanks for reporting this! I have confirmed that this bug is only affected by Ruby 2.5.1, and not other Ruby versions.  We are working on a fix internally now.

Would you mind posting your buildspec commands which set the BUNDLE environment variables here?  That would be helpful for anyone else running into this issue until we get the fix out.

**Comment by rcousens** [comment]
```
version: 0.2
env:
  variables:
    BUNDLE_PATH: "/usr/local/bundle"
    BUNDLE_APP_CONFIG: "/usr/local/bundle"
    BUNDLE_BIN: "/usr/local/bundle/bin"

phases:
  install:
    commands:
      - bundle install
```

**Comment by rcousens** [comment]
Would take me 30 seconds to make the changes and submit a PR so I'll do

**Comment by rcousens** [comment]
PR available as #65

**Comment by subinataws** [comment]
Fixed in https://github.com/aws/aws-codebuild-docker-images/releases/tag/1.3.2.0
